### ISSUE TITLE
Test failures

### ISSUE THREAD

**Issue by benji1304** [jongpie/NebulaLogger#253]
### Package Edition of Nebula Logger

Unlocked Package

### Package Version of Nebula Logger

v4.6.10	

### New Bug Summary

I'm getting 2 test failures from Gearset.

```
Tests passed: 441/443
Code coverage: 91%


Test Failures (2)

Test: LogEntryHandler_Tests.it_should_set_apex_class_details_for_inner_class
Message: System.AssertException: Assertion Failed: Expected: v52.0, Actual: v46.0

Test: LogEntryHandler_Tests.it_should_set_apex_class_details_for_top_level_class
Message: System.AssertException: Assertion Failed: Expected: v52.0, Actual: v46.0
```
![image](https://user-images.githubusercontent.com/7213889/143091468-6acb6a1e-f61c-4551-a1ef-4f3b50e6c623.png)

**Comment by jongpie** [comment]
Hi @benji1304 - I've been trying to reproduce this issue in a scratch org, but so far, I have not been able to do so. Have you made any modifications to the Logger metadata in your org? And just to confirm - are you using the unlocked package, or have you deployed the metadata directly to your org (using Gearset)?

**Comment by benji1304** [comment]
Hey @jongpie. No changes at all, not even looked at the metadata. Using the unlocked package. And currently Gearset is only for backup, nothing being deployed.

**Comment by jongpie** [comment]
@benji1304 cool, thanks for confirming! I'll do some more testing on my side to see if I can reproduce/fix the issue - I'll keep you updated on my progress.

**Comment by benji1304** [comment]
hey @jongpie i just ran the LogEntryHandler_Tests directly in the dev console, and I get the same error. Nothing related to Gearset, it was just Gearset that notified me.

**Comment by SimonWhite** [comment]
@benji1304 do you by any chance use Nebula Core package?

I figured the test fails with the API version of nebc.Logger class.

**Comment by benji1304** [comment]
@SimonWhite I'm using the unlocked package v4.6.11 - i don't think that's the core package?

**Comment by SimonWhite** [comment]
@benji1304 I thought maybe you have [this package](https://bitbucket.org/nebulaconsulting/nebula-core/src/master/) installed in the org (I think this is the reason why tests are failing in my org).
Or if there are any other classes named Logger.

**Comment by benji1304** [comment]
@SimonWhite hm - not sure. I used the link on from the repo?
![image](https://user-images.githubusercontent.com/7213889/145901414-1843f6b5-30f7-45bb-adae-72393a8f2723.png)

**Comment by jongpie** [comment]
@benji1304 I was finally able to reproduce - and fix - this issue. I don't have a timeline for the next release yet, but this will be resolved in the next release, `v4.6.15`. Thanks again for reporting this!

And @SimonWhite, thanks for your input! Your theory of "multiple packages containing a `Logger` class" proved to be very helpful - I was able to use that theory to find the root cause of the issue. Thanks so much for your help!

**Comment by benji1304** [comment]
Fantasic! thanks both @jongpie and @SimonWhite

**Comment by jongpie** [comment]
@benji1304 I've just released [v4.6.15](https://github.com/jongpie/NebulaLogger/releases/tag/v4.6.15), which includes a fix for this issue - let me know if you still run into any issues!
